$OpenBSD: patch-apertium-nno_nno_rlx,v 1.1 2018/10/17 04:39:54 bentley Exp $

Fix error with apertium-validate-modes 3.5.2:
"stand-alone o or O doesn't make sense - maybe you meant 0?"

From upstream 404dcb612d15c5c6ab2f3dfde7160ea9024ec8f8.

Index: apertium-nno.nno.rlx
--- apertium-nno.nno.rlx.orig
+++ apertium-nno.nno.rlx
@@ -6987,7 +6987,7 @@ REMOVE:mwe-verb-unlikely-adj ("<.* .*>"r adj pp); # e.
 
 #:6072
  "<vita>"   REMOVE:6072   (n nt) IF
-	 (O inf)
+	 (0 inf)
 ;
 #sjeldent i fleirtal - og eintal
 
